ecl/hqlcpp/hqlsource.cpp

```diff
@@ -1940,7 +1943,10 @@ ABoundActivity * SourceBuilder::buildActivity(BuildCtx & ctx, IHqlExpression * e
     if (nameExpr && nameExpr->queryValue())
     {
         if (isSpill)
-            graphLabel.append("\nSpill");
+        {
+            if (activityKind != TAKspillread)
+                graphLabel.append("\nSpill");
```

Comment:
Is this here so that diskcount and diskexists on spills are post-fixed to the graph label?

Reply:
yes

Reply:
yes